conllulex/lexcatter.py

```diff
@@ -16,6 +17,9 @@ def supersenses_for_lexcat(lc):
             }, lc  # PARSEME 1.1 verbal MWE subtypes
         return VSS
     if lc in ("P", "PP", "INF.P"):
+        PSS.add('p.Focus')  # to revisit - version 2.7
+        PSS.add('p.`d')  # to revisit - version 2.7
+        PSS.add('p.`i')  # to revisit - version 2.7
```

Comment:
this will modify PSS globally--I think you want:

```
return PSS | {'p.Focus', 'p.`d', 'p.`i'}
```

Reply:
OK, pushed new changes. Wouldn't the PSS object in the session be specific to the language, anyways? Since the session may currently be working with just one language at a time